File: python/raindrops/raindrops.py
from enum import IntEnum


class Drops(IntEnum):
    Pling = 3
    Plang = 5
    Plong = 7


def raindrops(number: int) -> str:
    """Convert a number to raindrop-speak."""
    drops = [drop.name for drop in Drops if not number % drop.value]
    return ''.join(drops) or str(number)

File: swift/raindrops/Sources/Raindrops/Raindrops.swift
struct Raindrops {
    let drops: Int
    
    init(_ drops: Int) {
        self.drops = drops
    }
    
    var sounds: String {
        var raindropSpeak = String()
        
        if drops % 3 == 0 { raindropSpeak += "Pling" }
        if drops % 5 == 0 { raindropSpeak += "Plang" }
        if drops % 7 == 0 { raindropSpeak += "Plong" }
        
        if raindropSpeak.isEmpty { raindropSpeak = String(drops)}
        
        return raindropSpeak
    }
}
